challenge3.md

```diff
@@ -1,26 +1,31 @@
 # Deepx's Front-end Challenge
 
 ## Description
-The challenge is to build a quick UI for listing and marking github issues as favorites.
+
+The challenge is to build a front-end application using HTML/CSS and any language/framework you want for listing GitHub issues and marking them as favorites.
+
+You are free to use any stack (language/framework) you want (like React, Angular, Vue, Elm, etc) and we will also evaluate your knowledge about the technologies you selected for this challenge.
 
 ## Requirements
-There are 4 requirements:
-- The user must be able to list issues
-- The user must be able to sort either by id or state
-- The user must be able to favorite an issue.
-- The user must be able to navigate to another screen, this screen will have a favorite list where it will list all the companies that he favorites.
 
-The design is up to you!
+The application to be developed needs to have the basic four features:
+
+- List issues using the Github API.
+- Sort issues either by id or state.
+- Favorite issues.
+- Navigate to a different screen that will have all the issues that were favorited.
 
-Please write the code as if it would go to production right after you commit it!
+Please note that the intent of any challenge is not to 'solve the problem', but showcase good practices while solving the problem.
```

Comment:
- not *only* to solve the problem

remove the quotes

Reply:
Fixed, removed the quotes and added __only__